README.md

```diff
@@ -67,86 +84,58 @@ The table below outlines what version of Nexus Repository the plugin was built a
 
-There are a range of options for installing the R plugin. You'll need to build it first, and
-then install the plugin with the options shown below:
-
-### Temporary Install
-
-Installations done via the Karaf console will be wiped out with every restart of Nexus Repository. This is a
-good installation path if you are just testing or doing development on the plugin.
-
-* Enable Nexus console: edit `<nexus_dir>/bin/nexus.vmoptions` and change `karaf.startLocalConsole`  to `true`.
-
-  More details here: https://help.sonatype.com/display/NXRM3/Installing+Bundles
-
-* Run NXRM console mode (the below assumes you are running as 'nexus' user as recommended in our documentation) 
-and enter the bundle command after it has started:
-  ```
-  # sudo su - nexus
-  $ cd <nexus_dir>/bin
-  $ ./nexus run
-  > bundle:install file:///tmp/nexus-repository-r-1.0.3.jar
-  ```
-  This will return text similar to "Bundle ID: 282".  You can double check this ID using bundle:list 
-  (for org.sonatype.nexus.plugins:nexus-repository-r; it should be the last one).  Then start it using this 
-  bundle command:
-  ```
-  > bundle:start <org.sonatype.nexus.plugins:nexus-repository-r ID>
-  ```
-
-### (more) Permanent Install
-
-For more permanent installs of the nexus-repository-r plugin, follow these instructions:
-
-* Copy the bundle (nexus-repository-r-1.0.3.jar) into <nexus_dir>/deploy
+### Permanent Reinstall
 
-This will cause the plugin to be loaded with each restart of Nexus Repository. As well, this folder is monitored
-by Nexus Repository and the plugin should load within 60 seconds of being copied there if Nexus Repository
-is running. You will still need to start the bundle using the karaf commands mentioned in the temporary install.
-
-### (most) Permanent Install
-
-If you are trying to use the R plugin permanently, it likely makes more sense to do the following:
-
-* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-r/1.0.3/nexus-repository-r-1.0.3.jar`
+* Copy the new bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-r/1.1.1-SNAPSHOT/nexus-repository-r-1.1.1-SNAPSHOT.jar`
 * If you are using OSS edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
 * If you are using PRO edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
+
    ```
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
-   +     <feature version="1.0.3" prerequisite="false" dependency="false">nexus-repository-r</feature>
+   +     <feature version="1.1.1.SNAPSHOT" prerequisite="false" dependency="false">nexus-repository-r</feature>
```

Comment:
NOTE: Not sure if there's a way around it but it kind of sucks we've still bound ourselves to updating a version in a document even by putting the plugin in house=(

Reply:
This would be the last version update.